src/System.Data.SqlClient/tests/Tools/TDS/TDS.Servers/Microsoft.SqlServer.TDS.Servers.xml

```diff
@@ -0,0 +1,901 @@
+<?xml version="1.0"?>
```

Comment:
These xmls should not be included. They were a bug that I fixed in https://github.com/dotnet/maintenance-packages/pull/154

Can you please delete these xmls from your branch, rebase your branch on top of the latest bits in main, then build again to confirm the xmls are not showing up?